apps/condo/domains/marketplace/components/Invoice/BaseInvoiceForm.tsx

```diff
@@ -552,10 +555,12 @@ const ServicesList = ({ organizationId, propertyId, form, currencySymbol, disabl
             }
         }
 
-        marketItemGroups.sort((a, b) => a.order > b.order ? 1 : -1)
+        marketItemGroups
+            .sort((a, b) => a.label > b.label ? 1 : -1)
+            .sort((a, b) => a.order > b.order ? 1 : -1)
         marketItemGroups.forEach(group => {
             group.options
-                .sort((a, b) => a.key > b.key ? 1 : -1)
+                .sort((a, b) => a.labelForSort > b.labelForSort ? 1 : -1)
```

Comment:
Couldn't it be done on the server side?

Reply:
I think no. 
We need to display MarketPriceScopes sorted by MarketCategory (and parent category if it has) name and order fields (MarketPriceScope -> MarketItemPrice -> MarketItem -> MarketCategory).
We can add the column in MarketPriceScope and calculate it for sorting on backend, but i think it's overhead.